projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-form/address-form.component.html

```diff
@@ -156,13 +154,20 @@
       </div>
     </div>
   </div>
-  <div class="cx-checkout-btns row">
-    <div class="col-md-12 col-lg-6" *ngIf="showCancelBtn">
-      <button class="btn btn-block btn-action" (click)="back()">
+  <div class="cx-checkout-btns row col-md-12 col-lg-12">
+    <div class="col-md-12 col-lg-6 pl-0" *ngIf="showCancelBtn">
```

Comment:
Why are we overriding col paddings? Usually means bootstrap structure is not set right. Maybe those cols in row divs are a problem?

Reply:
This won't be needed when I remove `col-md-12 col-lg-12` classes from `<div class="cx-checkout-btns row">`